book/src/development/infrastructure/changelog_update.md

```diff
@@ -38,7 +38,7 @@ Usually you want to write the changelog of the **upcoming stable release**. Make
 sure though, that `beta` was already branched in the Rust repository.
 
 To find the commit hash, issue the following command when in a `rust-lang/rust`
-checkout:
+checkout `beta` branch:
```

Comment:
The above text already describes which branch you want to run this command on. So I would write something like

```suggestion
checkout (most of the time on the `upstream/beta` branch):
```